kinode/src/main.rs

```diff
@@ -3,19 +3,19 @@
 use anyhow::Result;
 use clap::{arg, value_parser, Command};
 use lib::types::core::*;
-#[cfg(feature = "simulation-mode")]
+//#[cfg(feature = "simulation-mode")]
 use ring::{rand::SystemRandom, signature, signature::KeyPair};
 use std::env;
 use std::sync::Arc;
 use tokio::sync::mpsc;
 
 mod eth;
+mod fakenet;
```

Comment:
Should this be behind a feature flag? In general, please test compiling this PR in both simulation-mode and regularly, and make sure there are no compiler warnings about unused code in either compilation

Reply:
I put it behind a flag for now even though I think there isn't a huge difference rn.

simulation-mode has a fair bit of previous and current unused variables in main.rs and register, we could do something like `#![cfg_attr(feature = "simulation-mode", allow(unused))]` in main.rs to surpress them..

Reply:
It's best to keep unused code flagged-out for maximum clarity